docs/install/_linux.rst

```diff
@@ -6,23 +6,32 @@ Debian/Ubuntu
 Compatibility
 -------------
 
-Debian/Ubuntu 16.04 and up - anything that's *not* end-of-life
-
+Debian 8 (Jessie), Debian 9 (Stretch), Ubuntu 16.04+ and up - anything that's *not* end-of-life. Ubuntu-based distributions count a number of flavours, for instance Xubuntu and Mint.
 
 .. _ppa:
 
 Install from PPA repository
 ---------------------------
 
-Use the following commands in `Terminal <https://help.ubuntu.com/community/UsingTheTerminal>`_ to add the PPA and install Kolibri:
+**In Ubuntu-based distributions**, use the following commands in `Terminal <https://help.ubuntu.com/community/UsingTheTerminal>`_ to add the PPA and install Kolibri:
 
 .. code-block:: bash
 
-    sudo apt-get install software-properties-common python-software-properties
+    sudo apt-get install software-properties-common dirmngr
     sudo add-apt-repository ppa:learningequality/kolibri
     sudo apt-get update
     sudo apt-get install kolibri
 
+**In Debian-based distributions** you need to use these commands to point to our Launchpad PPA:
+
+.. code-block:: bash
+
+    sudo apt-get install software-properties-common dirmngr
```

Comment:
I don't understand why this is needed if later the repository info is added manually. Either `software-properties-common` is needed to use `add-apt-repository` or it's not needed if the `/etc/apt/sources.list.d/*.list`file is created manually. 
Both things are not needed.

Reply:
You are right. This was just intended as a merge PR, but we could fix later. I think `dirmngr` is still needed for `apt-key`.

Reply:
Fixed in 5b4076c2ce8c72430bd69c3da8c1be0b9798a3c6

Reply:
> You are right. This was just intended as a merge PR, but we could fix later. I think `dirmngr` is still needed for `apt-key`.

Yes, it does, if `apt-key` is used, but not if `add-apt-repository ppa:learningequality/kolibri` is used.
I prefer to use apt-key as it's compatible for Raspberry, we don't have a ppa for RPi, so add-apt-repository won't work for this device.